documentation/source/simulator_support.rst

```diff
@@ -256,6 +256,32 @@ Support is preliminary.
 Noteworthy is that despite GHDL being a VHDL simulator, it implements the VPI interface.
 
 
+.. _sim-ghdl-waveforms:
+
+Waveforms
+---------
+
+To get waveforms in VCD format, set the :make:var:`SIM_ARGS` option to ``--vcd=anyname.vcd``,
+for example in a Makefile:
+
+.. code-block:: make
+
+    SIM_ARGS+=--vcd=anyname.vcd
+
+The option can be set on the command line, as shown in the following example.
+
+.. code-block:: bash
+
+    SIM_ARGS=--vcd=anyname.vhd make SIM=ghdl
+
+A VCD file named ``anyname.vcd`` will be generated in the current directory.
+
+The :make:var:`SIM_ARGS` option allows access to all
+[GHDL Command Line Run (-r) options](https://ghdl.readthedocs.io/en/latest/using/Simulation.html#using-simulation).
```

Comment:
Alternatively, you can add an intersphinx link (see https://github.com/ghdl/ghdl-cosim/blob/master/doc/conf.py#L132) and replace these references with:

```rst
:ghdl:`USING:Simulation` or :ghdl:`simulation_options`
:ghdl:`export_waves`
```

The syntax you are using here seems to be Markdown, although the document is written in reStructuredText.

Reply:
I changed to intersphinx links and correct FST/GHW case.

Reply:
Although CI is not complaining explicitly about it, I'd say you need to add the intersphinx mapping in https://github.com/jwrr/cocotb/blob/jwrr/ghdlwaves/documentation/source/conf.py#L49. Just copy the one I referenced above.

See https://cocotb--1980.org.readthedocs.build/en/1980/simulator_support.html?highlight=ghdl#sim-ghdl-waveforms and:

```
https://readthedocs.org/projects/cocotb/builds/11449536/

/home/docs/checkouts/readthedocs.org/user_builds/cocotb/checkouts/1980/documentation/source/simulator_support.rst:279: WARNING: Unknown interpreted text role "ghdl".
/home/docs/checkouts/readthedocs.org/user_builds/cocotb/checkouts/1980/documentation/source/simulator_support.rst:279: WARNING: Unknown interpreted text role "ghdl".
```

Reply:
I added ghdl to intersphinx_mapping in conf.py and got the links to use it after a few tries.

Reply:
Great! Thanks for your responsiveness. I know it is annoying at first to do so many iterations for an apparently simple contribution. However, as said, I think that tightly cross-referencing docs of different projects is a good excuse to build a community, given that most of the projects use Sphinx. Hence, I'm really glad that you used intersphinx.

Reply:
@umarcor thanks for walking me through the process.  Please comment on @eric-wieser latest suggestion that is brief and clear but loses some information about FST and GHW.  I think brief is probably better because clicking the link will provide all extra info.